packages/create-sitecore-jss/src/initializers/vue/index.ts

```diff
@@ -0,0 +1,50 @@
+﻿import path from 'path';
+import { prompt } from 'inquirer';
+import { prompts, VueAnswer } from './prompts';
+import { Initializer } from '../../common/Initializer';
+import { transform } from '../../common/steps';
+import { VueArgs } from './args';
+import { FetchWith } from '../../common/prompts/base';
+import chalk from 'chalk';
+
+export default class VueInitializer implements Initializer {
+  get isBase() {
+    return true;
+  }
+
+  async init(args: VueArgs) {
+    const defaults = args.yes
+      ? {
+          appName: 'sitecore-jss-vue',
+          hostName: 'https://cm.jss.localhost',
```

Comment:
I suppose that we should have similar value, right? https://github.com/Sitecore/jss/commit/3599eb695b6cf57d8501c0386b198c7ad96fd52e
@CobyPear

Reply:
Correct, `sitecore-jss-vue.dev.local`